moveit_ros/visualization/motion_planning_rviz_plugin/src/ui/motion_planning_rviz_plugin_frame.ui

```diff
@@ -1169,19 +1169,19 @@
            </spacer>
           </item>
           <item>
-           <widget class="QPushButton" name="export_scene_text_button">
+           <widget class="QPushButton" name="export_scene_geometry_text_button">
             <property name="text">
-             <string>&amp;Export Scene Objects</string>
+             <string>&amp;Export Objects</string>
             </property>
             <property name="toolTip">
              <string>Export scene geometry to a .scene file (only geometry information is preserved)</string>
             </property>
            </widget>
           </item>
           <item>
-           <widget class="QPushButton" name="import_scene_text_button">
+           <widget class="QPushButton" name="import_scene_geometry_text_button">
             <property name="text">
-             <string>&amp;Import Scene Objects</string>
+             <string>&amp;Import Objects</string>
```

Comment:
Here you even reduced the button text? I think, _scene_ should be mentioned.

Reply:
If you rely on the frame title "Scene Geometry", you can shorten the button text to "Import".

Reply:
True, why not. I'll do that.